packages/grafana-ui/src/components/Table/Table.story.tsx

```diff
@@ -99,21 +112,25 @@ const defaultThresholds: ThresholdsConfig = {
   mode: ThresholdsMode.Absolute,
 };
 
-export const Simple = () => {
+export const Simple: Story = (args) => {
```

Comment:
Maybe change to Basic as a convention?
```suggestion
export const Basic: Story = (args) => {
```